cmake/openenclave-config.cmake.in

```diff
@@ -22,6 +22,10 @@ find_dependency(Threads)
 add_executable(openenclave::oeedger8r IMPORTED)
 set_target_properties(openenclave::oeedger8r PROPERTIES IMPORTED_LOCATION ${OE_BINDIR}/oeedger8r)
 
+# Similarly, this is a shell script.
+add_executable(openenclave::oegdb IMPORTED)
+set_target_properties(openenclave::oegdb PROPERTIES IMPORTED_LOCATION ${OE_BINDIR}/oegdb)
```

Comment:
@andschwa
It is not clear what openenclave::oegdb target provides the user. There ought to be some documentation or sample that demonstrates the usecase.

Reply:
It isn't? It's just an executable 😆 I'll add to the docs.